rust/chains/hyperlane-cosmos/src/mailbox.rs

```diff
@@ -186,12 +237,15 @@ impl Mailbox for CosmosMailbox {
         metadata: &[u8],
     ) -> ChainResult<TxCostEstimate> {
         let process_message = ProcessMessageRequest {
-            process_message: ProcessMessageRequestInner {
+            process: ProcessMessageRequestInner {
                 message: hex::encode(RawHyperlaneMessage::from(message)),
                 metadata: hex::encode(metadata),
             },
         };
 
+        println!("process_message: {:?}", process_message);
+        println!("metadata: {:?}", metadata);
```

Comment:
ditto, remove or convert to logging library
these are also already `#[instrument`ed